rclpy/test/test_node.py

```diff
@@ -171,6 +178,88 @@ def test_node_names_and_namespaces(self):
         # test that it doesn't raise
         self.node.get_node_names_and_namespaces()
 
+    def assert_qos_equal(self, qos_profile, qos_dict):
+        # Depth and history are skipped because they are not retrieved.
+        self.assertEqual(qos_profile.durability, qos_dict['durability'],
+                         'Durability is unequal')
```

Comment:
nitpick: do not indent like this, indent only once each time and only ever four spaces.

This should be:

```python
        self.assertEqual(qos_profile.durability, qos_dict['durability'],
            'Durability is unequal')
```

Reply:
Done.